packages/eas-cli/src/utils/expoUpdatesCli.ts

```diff
@@ -24,13 +25,18 @@ export async function expoUpdatesCommandAsync(projectDir: string, args: string[]
   }
 
   try {
-    return (await spawnAsync(expoUpdatesCli, args)).stdout;
+    return (await spawnAsync(expoUpdatesCli, args, { stdio: 'pipe' })).stdout;
```

Comment:
This technically is not required here, because this `spawn` should default to `pipe`, right? Asking to confirm.

We're not using the `turtle-spawn` here that defaults to `inherit`:
https://github.com/expo/eas-build/blob/997d6e65c7fea16184d038fdf831352346c1685d/packages/turtle-spawn/src/index.ts#L16-L19

Reply:
Correct. I added it just to keep the code reasonably consistent between this and `build-tools`.